File: tools/dbench/answers/go_haiku/s2/m1_oldest.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	f, _ := os.Open("users.txt")
	defer f.Close()

	sc := bufio.NewScanner(f)
	var name string
	age := -1

	for sc.Scan() {
		parts := strings.Fields(sc.Text())
		if len(parts) != 3 {
			continue
		}
		a, err := strconv.Atoi(parts[2])
		if err != nil {
			continue
		}
		if a > age {
			age = a
			name = parts[1]
		}
	}

	fmt.Printf("%s %d\n", name, age)
}

File: tools/dbench/answers/python_sonnet/s3/m1_oldest.py
users = []
with open("users.txt") as f:
    for line in f:
        p = line.split()
        if len(p) == 3:
            try:
                users.append((p[1], int(p[2])))
            except ValueError:
                pass
name, age = max(users, key=lambda x: x[1])
print(f"{name} {age}")
